\documentclass[10pt]{article}
\usepackage{geometry}                % See geometry.pdf to learn the layout options. There are lots.
\geometry{letterpaper}                   % ... or a4paper or a5paper or ... 
\usepackage{graphicx}
\usepackage[hyphens]{url}
\usepackage{fancyhdr}
\usepackage{hyperref}
\usepackage{parskip}
\usepackage{helvet}
\usepackage[all]{nowidow}

\renewcommand{\familydefault}{\sfdefault}
\renewcommand{\arraystretch}{2}
\hypersetup{backref,  urlcolor=blue, colorlinks=true, citecolor=black, linkcolor=black, hyperindex=true}
\setlength{\parskip}{3pt}

\title{Brief Article}
\author{The Author}
%\date{}                                           % Activate to display a given date or no date
\pagestyle{fancy}
\lhead{\bf \large Karen A. Cranston} 

\begin{document}
\noindent Department of Biology \hfill \url{https://twitter.com/kcranstn} \\
Duke University  \hfill \url{http://kcranston.github.io} \\
Durham, NC, USA \hfill  \url{http://github.com/kcranston} \\
karen.cranston@gmail.com  \\

\vspace{0.4cm}
I am an evolutionary computational biologist who is passionate about how computational tools and skills facilitate the adoption and practice of open science. I am the lead PI of Open Tree of Life (\url{http://opentreeoflife.org}), which synthesizes an online, constantly updating tree of life based on community curation of published evolutionary trees. 

\section*{Skill summary}

\emph{Technical}: extensive experience with python (web applications, libraries, scripting); also knowledge of database (SQL) design; API design / implementation; Ansible deployment; C / C++, UI design \\

\emph{Outreach / education}: teaching workshops, developing course material, coordinating programs, and evidence-based instructor training; focus in on teaching technical and analytical skills to researchers; co-organized several hackathons to build tools and communities in open-source evolutionary informatics.\\ 

\emph{Research}: phylogenetic methods, including gene tree inference, vizualization, supertrees; statistical genetics, including Bayesian Markov Chain Monte Carlo\\

\section*{Education}
{\bf PhD} (2007) Statistical Genetics. Department of Medical Genetics, University of Alberta, Edmonton, AB, Canada. 

{\bf B.Sc., Honours, First Class} (1996), Genetics. University of Manitoba, Winnipeg, MB, Canada \\

% Education
\section*{Employment}

\noindent \textbf{Research Scientist} \\
2015-present, Department of Biology, Duke University, Durham, NC \\
Lead PI of Open Tree of Life (\url{http://opentreeoflife.org}) project that summarizes published phylogenetic trees into a synthetic tree of life; designing and writing software for storing, accessing and analyzing phylogeny data; leading hackathons and workshops; managing overall project, including software development.

\noindent \textbf{Training Coordinator and Bioinformatics Project Manager} \\
2010-2015, National Evolutionary Synthesis Center (NESCent), Durham, NC. \\
Management of informatics support for a broad range of evolutionary science projects; coordination of training programs in informatics and analytical skills for scientists; community building in open-source biodiversity and evolutionary informatics through hackathons and Google Summer of Code. \\

\noindent \textbf{Postdoctoral Research Associate} \\
2009-2010, Biodiversity Synthesis Center, The Field Museum, Chicago, IL  \\
Interactive visualization of taxonomies and large phylogenies with Encylopedia of Life and iPlant Collaborative; summarizing large-scale gene tree incongruence across genomes \\

\noindent \textbf{Postdoctoral Research Associate} \\
2007-2009, Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ \\
PhyLoTA broswer, summarizing phylogenetic signal in GenBank; phylogenomic analyses of wild and domesticated rice species \\ 

\noindent \textbf{Laboratory technician} \\
1997-2002, Cangene Corporation, Winnipeg, MB. \\
Maintenance of cell cultures; design and implementation of protein chemistry assays; analytical method validation. \\

\noindent \textbf{Technical writer}
1996-1997, Integrated Engineering Software, Winnipeg, Canada, 1996-1997 \\
Write technical documentation and online help systems for engineering simulation software. \\

%Pubs
%\vspace{2mm}
\section*{Publications}

I have 16 peer-reviewed scientific papers, and 7 commentaries / preprints (noted with *).

\subsection*{Phylogenetic methods}
Hinchliff, C. E., et al., (2015). Synthesis of phylogeny and taxonomy into a comprehensive tree of life. Proceedings of the National Academy of Sciences, 112(41), 12764-12769. \url{http://dx.doi.org/10.1073/pnas.1423041112}. \textit{In the Altmetrics Top 100 for 2015 and in top 5\% overall; 53 news reports; 832 Tweets, 14 blogs; https://www.altmetric.com/details/4529894}

Evans, M. E., et al., (2011). Extreme environments select for reproductive assurance: evidence from evening primroses (Oenothera). \textit{New Phytologist}, 191(2), 555-563. \url{http://dx.doi.org/10.1111/j.1469-8137.2011.03697.x} 

Ammiraju, J. S., et al., (2010). Spatiotemporal patterns of genome evolution in allotetraploid species of the genus Oryza. \textit{The Plant Journal}, 63(3), 430-442. \url{http://dx.doi.org/10.1111/j.1365-313X.2010.04251.x} 

Cranston, K. A. (2010) Quantifying gene tree incongruence across varying phylogenetic depths. In L. Lacey Knowles, Laura S. Kubatko (Eds.) \textit{Estimating Species Trees: Practical and Theoretical Aspects}, Wiley-Blackwell.

Cranston, K. A., et al., (2010). Phylogenomic analysis of BAC-end sequence libraries in Oryza (Poaceae). Systematic botany, 35(3), 512-523. \url{http://dx.doi.org/10.1600/036364410792495872} 

Cranston, K. A., et al., (2009). Species trees from highly incongruent gene trees in rice.  \textit{Systematic Biology} 58(5), 489-500. \url{http://dx.doi.org/10.1093/sysbio/syp054} 

Cranston, K. A., \& Rannala, B. (2007). Summarizing a posterior distribution of trees using agreement subtrees. \textit{Systematic Biology} 56(4), pp. 578-590. \\
\url{http://dx.doi.org/10.1080/10635150701485091} 

Cranston, K., \& Rannala, B. (2005). Closing the gap between rocks and clocks. Heredity, 94(5), 461-462. \url{http://dx.doi.org/10.1038/sj.hdy.6800644}

\subsection*{Software and databases}
McTavish, E. J., et al., (2015). Phylesystem: a git-based data store for community curated phylogenetic estimates. Bioinformatics, btv276. \url{http://dx.doi.org/10.1093/bioinformatics/btv276} 

Ksepka, D.T., et al., (2015) The Fossil Calibration Database, A New Resource for Divergence Dating. \textit{Systematic Biology}. \url{http://dx.doi.org/10.1093/sysbio/syv025}

Stoltzfus, A., et al., (2013). Phylotastic! Making tree-of-life knowledge accessible, reusable and convenient. \textit{BMC bioinformatics}, 14(1), 158. \url{http://dx.doi.org/10.1186/1471-2105-14-158} 

Sanderson, M. J., et al., (2008). The PhyLoTA Browser: processing GenBank for molecular phylogenetics research. \textit{Systematic Biology} 57(3), pp. 335-346. \url{http://dx.doi.org/10.1080/10635150802158688} 

\subsection*{Open science, Collaboration, and Education}
* Wilson, G., et al., (2016). Good Enough Practices in Scientific Computing. arXiv preprint \url{http://arxiv.org/abs/1609.00037}

Katz, D.S. et al., (2016). Report on the Second Workshop on Sustainable Software for Science: Practice and Experiences (WSSSPE2). Journal of Open Research Software. 4(1), p.e7. \url{http://doi.org/10.5334/jors.85}

Teal, T. K., et al., (2015) Data Carpentry: Workshops to Increase Data Literacy for Researchers. \textit{International Journal of Digital Curation}. 10(1):135-143. \url{http://dx.doi.org/10.2218/ijdc.v10i1.351}
 
Cranston, K. A., et al., (2014) Best Practices for Data Sharing in Phylogenetic Research. \textit{PLOS Currents Tree of Life}. Jun 19. Edition 1. \\
\url{http://dx.doi.org/10.1371/currents.tol.bf01eff4a6b60ca4825c69293dc59645} 

* Katz, D. S., et al., (2014). Second Workshop on on Sustainable Software for Science: Practice and Experiences (WSSSPE2): Submission, Peer-Review and Sorting Process, and Results. arXiv preprint \url{http://arxiv.org/abs/1411.3464}.

* Vision, T., \& Cranston, K. A. (2014) Open data for evolutionary synthesis: an introduction to the NESCent collection. Nature Scientific Data. \url{http://dx.doi.org/10.1038/sdata.2014.30}

* Cranston, K. A., et al., (2014): Simple rules for sharing phylogenetic data. figshare. \url{http://dx.doi.org/10.6084/m9.figshare.997763}

Rodrigo, A., et al., (2013). Science incubators: synthesis centers and their role in the research ecosystem. PloS biology, 11(1), e1001468. \url{http://dx.doi.org/10.1371/journal.pbio.1001468} 

* Rees, J. A., et al., (2013): Response to GBIF request for consultation on data licenses. figshare. \url{http://dx.doi.org/10.6084/m9.figshare.799766}

* Cranston, K. A., et al., (2013): A grassroots approach to software sustainability. figshare.
\url{http://dx.doi.org/10.6084/m9.figshare.790739}

Goff, S. A., et al., (2011). The iPlant collaborative: cyberinfrastructure for plant biology. \textit{Frontiers in plant science}, 2. \url{http://dx.doi.org/10.3389/fpls.2011.0003} 

\section*{Education and outreach}

\begin{itemize}
\item{NESCent Academy: coordinated series of hands-on summer workshops in Evolutionary Biology and Informatics, 2010-2014.}
\item{Software / Data Carpentry: certified instructor since 2013; working towards Instructor Training certification in 2017; co-lead development of reproducible science curriculum}
\item{Phylogenetics workshops: taught phylogenetic methods and software in workshops at Wuhan Institute of Virology, Fudan University, Kenyan Medical Research Centre, The Field Museum, and Bodega Bay Marine Laboratory} 
\item{Hackathons: co-organized four hackathons on evolutionary informatics through NESCent and Open Tree of Life; see \url{https://informatics.nescent.org/wiki/Main_Page}}
\end{itemize}

% Other 'lookin good' stuff
\section*{Professional activities}
\begin{itemize}
\item{Board of Directors: Open Bioinformatics Foundation (\url{https://open-bio.org/}), 2015-present}
\item{Data Carpentry founding board member (\url{http://datacarpentry.org}), 2014-present}
\item{Organization administrator for NESCent Google Summer of Code participation, 2011-2013}
\item{Program committee / panelist, Workshop on Sustainable Software for Science: Practice and Experiences (WSSSPE), \url{http://wssspe.researchcomputing.org.uk}, 2013, 2014, 2016}
\item{Society of Systematic Biology council, 2012-2014}
\item{Systematic Biology Editorial Board, 2011-present}
\item{Leadership team, Informatics for Evolutionary Biology (iEvoBio, \url{http://ievobio.org}) conference, 2013-2016}
\item{Phyloinformatics Research Foundation board member, 2010-present}
\item{Team lead, iPlant Tree Of Life working group on Tree Visualization, 2009-2012}
\item{Invited participant: iPlant Collaborative (\url{http://iplantcollaborative.org}) Grand Challenge Workshop on Assembling the Tree of Life to Enable the Plant Sciences, November 2009, Biosphere 2, Oracle, AZ}
\item{Journal referee for: Systematic Biology, Trends Ecology and Evolution, PeerJ, Nucleic Acids Research, Journal of Open Research Software, PLOS Computational Biology, Molecular Biology and Evolution}. See \href{https://publons.com/a/213683/}{Publons reviewer profile} for recent activity. 
\item{National Science Foundation panelist, 2011 - 2014}
\end{itemize}

\end{document}  
